.github/workflows/snyk-security-scan.yml

```diff
@@ -7,6 +7,14 @@ jobs:
   synk-security-scan:
     name: Snyk security scan
     runs-on: ubuntu-latest
+    # see this PR regarding the permissions needed for this workflow
+    # https://github.com/snyk/actions/pull/79
+    permissions:
+      # required for all workflows
+      security-events: write
+      # only required for workflows in private repositories
+      actions: read
+      contents: read
```

Comment:
fixes this issue

https://github.com/grafana/oncall/actions/runs/9467296116/job/26081010552?pr=4502#step:6:13

Reply:
scratch that, still not working.. opened a separate issue to fix this

https://github.com/grafana/oncall/issues/4503